exporter/signalfxexporter/translation/translator.go

```diff
@@ -524,11 +526,7 @@ func aggregateDatapoints(
 	// group datapoints by dimension values
 	dimValuesToDps := make(map[string][]*sfxpb.DataPoint, len(dps))
 	for i, dp := range dps {
-		aggregationKey, err := getAggregationKey(dp.Dimensions, dimensionsKeys)
-		if err != nil {
-			logger.Debug("datapoint is dropped", zap.String("metric", dp.Metric), zap.Error(err))
-			continue
-		}
+		aggregationKey := getAggregationKey(dp.Dimensions, withoutDimensions)
```

Comment:
Would it be helpful for readers to have a comment here that says `aggregationKey` looks like `dim1:val1//dim2:val2`?

Reply:
added a comment to the function